regression/goto-analyzer/ternary-operator-intervals/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--variable-sensitivity --vsd-values intervals --show
+^EXIT=0$
+^SIGNAL=0$
+main::1::b1 \(\) -> \[1, 1\] @ \[2\]
+main::1::b2 \(\) -> \[0, 0\] @ \[3\]
+main::1::b3 \(\) -> \[0, 1\] @ \[5\]
+main::1::i \(\) -> \[A, A\] @ \[7\]
```

Comment:
I realise this is probably not your doing but if the output is in hex, it should have a leading `0x`.

Reply:
It did take me a little while to realise that the interval output was hex. I would be delighted to change everything to decimal or everything to hex :)

Reply:
All decimal or all hex (or even, all both) are valid design choices and arguable.  The current mix and lack of `0x` is definitely wrong.  I would be very happy to have PRs that fix this.  Again, out of scope for this PR.